config/config.json

```diff
@@ -0,0 +1,13 @@
+{
+    "database" : {
+        "port": "27017",
+        "uri": "localhost",
+        "collection": "PokeData"
+    },
+    "twitter" : {
+        "consumer_key": "46yqEVwOOkf7dtN5Zfqxw4Nzu",
```

Comment:
@samitsv sure, I just asked as the comment said it's revoked and I could still find it, so was misled.